### ISSUE TITLE
##  Feature Request: Add a `requirements.txt` and backend folder scaffold for the Python detection service

### ISSUE THREAD

**Issue by prince-pokharna** [ShubhangiRoy12/MooVit#919]
### Description
The README documents a full backend stack (`Flask`/`FastAPI`, `YOLOv8/v11/v12`, `OpenCV`, `TensorFlow`/`PyTorch`) with an installation step:
cd backend
pip install -r requirements.txt
python app.py
However, there is no `backend/` directory, no `app.py`, and no `requirements.txt` anywhere in the repository. The detection-related folders that do exist (`live-detection`, `sharp-detection`) appear to be empty or placeholder directories. This means the core AI/safety functionality described as MooVit's main value proposition is currently **not buildable** by a contributor following the docs.

### Proposed Scope
- Create `backend/app.py` as a minimal Flask entrypoint
- Add `backend/requirements.txt` pinning `flask`, `opencv-python`, `ultralytics` (for YOLOv8), `numpy`
- Add a `/detect` POST endpoint accepting an image and returning bounding boxes + labels as JSON
- Wire the existing `live-detection` folder's intent into this scaffold

### Acceptance Criteria
- [ ] `backend/` folder exists with a runnable Flask app
- [ ] `pip install -r requirements.txt && python app.py` works per the README instructions
- [ ] At least one working `/detect` endpoint returns a JSON response for a test image

**Labels:** `enhancement`, `backend`, `help wanted`, `priority: high`

**Comment by Neelu-Verma-26** [comment]
Hi @prince-pokharna,

I'd like to work on this issue as part of GSSoC'26.

I've gone through the problem statement and acceptance criteria. I understand that this involves creating the backend scaffold, setting up the Flask entry point, adding the required dependencies, and ensuring the detection service follows the documented setup.

If this issue is still available, I'd be happy to work on it. Please assign it to me.

Thank you!

**Comment by aishwaryagangaraj-web** [comment]
Hi @prince-pokharna,

I'm interested in working on this feature as well. It aligns with my experience in Python, Flask, and computer vision.

I noticed another contributor has already requested assignment. If the issue becomes available or you decide to allow multiple contributors, I'd be happy to work on it and submit a PR.

Thank you!

**Comment by ParidhiMis** [comment]
Hi I am a GSSoC Contributor and I would like to work on this issue
/claim /assign
